### ISSUE TITLE
Students should see error message of failed compilation

### ISSUE THREAD

**Issue by thehappyhippo** [troeger/opensubmit#79]
Students get Emails about failed compilation but can not fix it, because they don't about the problem make has. In combination with https://github.com/troeger/opensubmit/issues/68 I vote for separate script for each process on a submission: compile, validate, full test.

**Comment by troeger** [comment]
I don't understand how the title of the issue relates to your description.

Compilation output is stored in the database and shown to the student, whether compilation fails or not. If you don't want the executor to call "make", don't activate the compilation feature for the assignment.

BTW, you should make your proposal for separate compilation scripts a dedicated "new feature" issue, instead of hiding it in #68 and #79. I like the idea, but it needs some good planning. Finding short-term solutions for problems like the one here is different from major feature additions.

So, in short: What is the problem here? How can it be reproduced? Is there a test case for that? If not, can you write one and contribute it?

**Comment by thehappyhippo** [comment]
Yes it was https://github.com/troeger/opensubmit/issues/77.

**Comment by troeger** [comment]
OK, than this is a duplicate. If you want compile script, please make this a separate new-feature issue.
